src/hooks/useEagerlyConnect.ts

```diff
@@ -18,15 +19,26 @@ async function connect(connector: Connector) {
 }
 
 export default function useEagerlyConnect() {
+  const dispatch = useAppDispatch()
+
   const selectedWalletBackfilled = useAppSelector((state) => state.user.selectedWalletBackfilled)
   const selectedWallet = useAppSelector((state) => state.user.selectedWallet)
 
+  let selectedConnection: Connection | undefined
+  if (selectedWallet) {
+    try {
+      selectedConnection = getConnection(selectedWallet)
+    } catch {
```

Comment:
this is unrelated to the fortmatic removal right? just another improvement

Reply:
it is related because if FORTMATIC is the saved option this all will error out